inference-engine/ie_bridges/java/sample/ArgumentParser.java

```diff
@@ -22,41 +22,31 @@ public String get(String flag) {
         if(res == null) {
             throw new RuntimeException("Wrong argument with flag: " + flag);
         }
-
         return res;
     }
 
-    public String get(String flag, String ifNull) {
+    public String get(String flag, String defaultValue) {
         String res = input.get(flag);
-
-        if(res == null) {
-            return ifNull;
-        } else {
-            return res;
-        }
-    }
-
-    public String getOrNull(String flag) {
-        return input.get(flag);
+        return (res != null) ? res : defaultValue;
     }
 
     public int getInteger(String flag) {
             return Integer.parseInt(get(flag));
     }
 
-    public int getInteger(String flag, int ifNull) {
-        String res = getOrNull(flag);
-        if(res != null)
-            return Integer.parseInt(res);
-        else
-            return ifNull;
+    public int getInteger(String flag, int defaultValue) {
+        try {
+            return Integer.parseInt(get(flag));
+        } catch (RuntimeException e) {
+            return defaultValue;
+        }  
```

Comment:
I don't think that's good to return default value in case of invalid argument - better
```java
String res = get(flag);
return (res != null) ? Integer.parseInt(res) : defaultValue;
```

Reply:
But then I need one more method ```get``` which doesn't throw an exception.